manifests/g/Glarysoft/GlaryUtilities/6.6.0.9/Glarysoft.GlaryUtilities.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.3.2 $debug=NVS1.CRLF.7-4-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Glarysoft.GlaryUtilities
+PackageVersion: 6.6.0.9
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: machine
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
+UpgradeBehavior: install
+ReleaseDate: 2024-02-05
+Installers:
+- Architecture: x86
+  InstallerUrl: https://download.filepuma.com/files/system-utilities/glary-utilities/Glary_Utilities_v6.6.0.9.exe
```

```suggestion
  InstallerUrl: https://download.glarysoft.com/gusetup.exe
```

Hi @c2y5,

The `PackageUrl` leads to this `InstallerUrl` - should it be used instead? I couldn't find that `PackageUrl` on the package's website.